modules/creating-oci-infra-resources-services.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_oci/installing-oci-agent-based-installer.adoc [Using the Agent-based Installer to install a cluster on OCI]
+
+:_mod-docs-content-type: PROCEDURE
+[id="creating-oci-infra-resources-services_{context}"]
+= Creating OCI infrastructure resources and services
+
+Before you install {product-title} on {oci-first}, you must create an {oci} environment on your virtual machine (VM) shape or bare-metal shape. By creating this environment, you can install {product-title} and deploy a cluster on infrastructure that supports a wide range of cloud options and strong security policies.
+
+.Prerequisites
+* You have prior knowledge of {oci} components. See link:https://docs.oracle.com/en-us/iaas/Content/GSG/Concepts/concepts.htm[Learn About Oracle Cloud Basics] in the Oracle documentation.  
+* You completed the link:https://docs.google.com/forms/d/e/1FAIpQLSdQHmw0BUWjQxPKC5_G7lk2_o9Pcnn2ON84Al6s2Ir254gv6Q/viewform[Request Access to Red Hat OpenShift on {oci} in Developer Preview] form. 
```

Comment:
Hi @pawanpinjarkar . Is this form still required? Any alternatives or do customers have direct access to the feature?

Reply:
I believe it needs to be present but please confirm with Ramon @racedo